### PR TITLE
Task dl/j244 add home screen

### PR DESCRIPTION
**Pull request by Denidzio**
image
image

### CODE REVIEW

File: src/screens/home/components/popularSingleCard/styles.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { ImageStyle, StyleSheet, TextStyle, ViewStyle } from 'react-native'
+
+type Styles = {
+  container: ViewStyle
+  background: ImageStyle
+  overlay: ViewStyle
+  author: TextStyle
+  title: TextStyle
+}
```
Comment (vitalik-prog) [inline src/screens/home/components/popularSingleCard/styles.ts:9]: Is this really necessary hard-typing styles? Maybe it redundant?

Comment (Denidzio) [reply src/screens/home/components/popularSingleCard/styles.ts:9]: > Is this really necessary hard-typing styles? Maybe it redundant?

It's convenient to track which styles are compatible with particular component

File: src/screens/home/components/recentlyPlayedCard/RecentlyPlayedCard.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react'
```
Comment (eugeneprib) [inline src/screens/home/components/recentlyPlayedCard/RecentlyPlayedCard.tsx:1]: The folder of the component also needs to be capitalized

File: src/screens/home/components/userAvatar/styles.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    backgroundColor: '#EEE',
+    overflow: 'hidden'
+  },
+  image: {
+    width: '100%',
+    height: '100%'
+  }
+})
+
+export { styles }
```
Comment (eugeneprib) [inline src/screens/home/components/userAvatar/styles.ts:24]: Usually styles are exported with default export

**Review by eugeneprib — CHANGES_REQUESTED** [review]

File: src/screens/home/components/recentlyPlayedCard/RecentlyPlayedCard.tsx (hunk 2 above)
Comment (eugeneprib) [reply src/screens/home/components/recentlyPlayedCard/RecentlyPlayedCard.tsx:1]: Screen containers included too

**Review by eugeneprib — CHANGES_REQUESTED** [review]

**Pull request closed without merging**